### PR TITLE
Create component TableControl

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Create component TableControl

image

### CODE REVIEW

File: examples/practice-typescript/src/components/TableController/index.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Button from 'components/Button';
+import InputField from 'components/Input';
+import './index.scss'
+
+const TableController = () => {
+  return (
+    <div className='table-control'>
+      <InputField
+        icon={require('../../assets/images/managers/icon-search.png')}
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/components/TableController/index.tsx]: use ES6 syntax `import` instead

File: examples/practice-typescript/src/components/TableController/index.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Button from 'components/Button';
+import InputField from 'components/Input';
+import './index.scss'
+
+const TableController = () => {
+  return (
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/components/TableController/index.tsx]: shorthand if possible

File: examples/practice-typescript/src/components/TableController/index.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableController/index.tsx]: Fixed

File: examples/practice-typescript/src/components/TableController/index.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableController/index.tsx]: Fixed

File: examples/practice-typescript/src/images.d.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+declare module '*.png' {
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/images.d.ts]: Are u need this?

File: examples/practice-typescript/src/components/TableController/index.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+.table-control {
+  display: flex;
+  z-index: 4;
+  background-color: $color-text-secondary;
+  justify-content: end;
+  align-items: center;
+  width: 100%;
+  padding: 1.15rem;
+  position: sticky;
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/TableController/index.scss]: Why sticky here? What if I have another content before this table, this style will not suitable.

Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableController/index.scss]: Fixed

File: examples/practice-typescript/src/images.d.ts (hunk 3 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/images.d.ts]: Fixed

File: examples/practice-typescript/src/components/TableController/index.scss [hunk 5]
```diff
@@ -0,0 +1,11 @@
+@import '../../styles/bases/index.scss';
+
+.table-control {
+  display: flex;
+  background-color: $color-text-secondary;
+  justify-content: end;
+  align-items: center;
+  width: 100%;
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/TableController/index.scss]: Do you need this?

File: examples/practice-typescript/src/components/TableController/index.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import Button from 'components/Button';
+import InputField from 'components/Input';
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/TableController/index.tsx]: Component name should same as file `Input`

File: examples/practice-typescript/src/components/TableController/index.scss (hunk 5 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableController/index.scss]: Fixed

File: examples/practice-typescript/src/components/TableController/index.tsx (hunk 6 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableController/index.tsx]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**